semantic-ui-react-kitchensink/src/main/scala/demo/Main.scala

```diff
@@ -0,0 +1,13 @@
+package demo
+
+import org.scalajs.dom
+import slinky.web.ReactDOM
+
+import scala.scalajs.js
+import scala.scalajs.js.annotation.JSImport
+
+object Main {
```

Comment:
not really for this PR, but would it be worthwhile to inline this and trim the demos to typically one file only?

Reply:
Not sure, if you are coming straight from Slinky repos, the distinction between `Main` and `App` is alway there. Plus I would also that say having each component in a separate file is a good practice that we should encourage.